test/integration/sanHistoryTwitterData.js

```diff
@@ -0,0 +1,45 @@
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHandlesNullData,
+  assertNumberOfRecords
+} = require('../integration_helper.js')
+
+const {
+  slug,
+  from,
+  to,
+  days,
+  numberOfDays,
+  formatDate
+} = require('../setup.js')
+
+describe('SAN_HISTORY_TWITTER_DATA', () => {
+  const expected = { date: 'string', followersCount: 'number' }
+
+  const response = san.SAN_HISTORY_TWITTER_DATA(slug, from, to)
+  const headers = response[0]
+  const results = response[1]
+  console.log(response)
+  testFieldTypes(results, expected)
+  testHandlesNullData(
+    'fetchHistoryTwitterData',
+    san.SAN_HISTORY_TWITTER_DATA,
+    slug,
+    from,
+    to)
+
+  it('has proper headers', () => {
```

Comment:
Why don't we have a test that historic data is forbidden?

Reply:
Because it isn't a premium metric